data_science/test_data.json

```diff
@@ -0,0 +1,201 @@
+[{
```

Comment:
This is a test file. It belongs in the test directory along with all the other testing stuff.